.travis.yml

```diff
@@ -6,8 +6,7 @@ cache:
 jdk:
   - oraclejdk8
 
-after_success:
-  - mvn --settings .travis/settings.xml --batch-mode -DskipTests install deploy
+install: mvn --settings .travis/settings.xml --batch-mode -DskipTests install deploy
```

Comment:
it seems a bit redundant to deploy it twice for tags, and weird to deploy during the install phase instead of doing it in the deploy phase